shell/src/main/java/alluxio/cli/fsadmin/report/CapacityCommand.java

```diff
@@ -238,13 +239,33 @@ private void printShortWorkerInfo(List<WorkerInfo> workerInfoList) {
         int usedPercentage = (int) (100L * usedBytes / capacityBytes);
         usedPercentageInfo = String.format(" (%s%%)", usedPercentage);
       }
-      print(String.format(SHORT_INFO_FORMAT, info.getAddress().getHost(),
+      print(String.format(shortInfoFormat, info.getAddress().getHost(),
           info.getLastContactSec(), "capacity", FormatUtils.getSizeFromBytes(capacityBytes)));
-      print(String.format(SHORT_INFO_FORMAT, "", "", "used",
+      print(String.format(shortInfoFormat, "", "", "used",
           FormatUtils.getSizeFromBytes(usedBytes) + usedPercentageInfo));
     }
   }
 
+  /**
+   * Gets the info format according to the longest worker name.
+   * @param workerInfoList the worker info list to get info from
+   * @param isShort whether exists only one tier
+   * @return the info format for printing long/short work info
+   */
+  private String getInfoFormat(List<WorkerInfo> workerInfoList, boolean isShort) {
+    int maxWokerNameLength = workerInfoList.stream().map(w -> w.getAddress().getHost().length())
+        .max(Comparator.comparing(Integer::intValue)).get();
+    int firstIndent = 16;
+    if (firstIndent <= maxWokerNameLength) {
+      // extend first indent according to the longest worker name
+      firstIndent = Double.valueOf(Math.ceil(maxWokerNameLength * 1.2)).intValue();
```

Comment:
If the worker name is really long, the gap (length * 0.2) may be too big, better to use a fixed gap.